### PR TITLE
Add pull request template

### PR DESCRIPTION
**Pull request by sergefdrv**
This pull request makes it more convenient to create a new pull request.

### CODE REVIEW

**Review by ynamiki — APPROVED** [review]
Thanks!

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+NOTE: Please remove irrelevant parts of this template and check the contribution guideline before submitting.
+
+This pull request ...
+
+-or-
+
+This is a successor of #{PULL_REQUEST_ID} to address review comments.
+
+Summary of changes since last revision:
+
```
Comment (nhoriguchi) [inline .github/pull_request_template.md:10]: There's a trailing whitespace here. Please remove it.

Comment (sergefdrv) [reply .github/pull_request_template.md:10]: Wow, I wonder how did you notice? (And regret using GitHub web-editor.)

Comment (nhoriguchi) [reply .github/pull_request_template.md:10]: I sometimes look at diff in terminal with `git log -p`, then trailing spaces are highlighted, so I noticed. But I don't always do this, so we are just fortunate.

**Pull request closed without merging**